### ISSUE TITLE
Nearly 0 accuracy on valid dataset at lora finetune

### ISSUE THREAD

**Issue by zxree** [espnet/espnet#6108]
When I was doing lora fine-tuning, I noticed that the accuracy of my training set kept improving, but the accuracy of the validation set stayed the same and close to 0. Here's my config file：

normalize: null

encoder: whisper
encoder_conf:
    whisper_model: medium
    dropout_rate: 0.0
    use_specaug: true
    specaug_conf:
        apply_time_warp: true
        time_warp_window: 5
        time_warp_mode: bicubic
        apply_freq_mask: true
        freq_mask_width_range:
        - 0
        - 40
        num_freq_mask: 2
        apply_time_mask: true
        time_mask_width_ratio_range:
        - 0.
        - 0.12
        num_time_mask: 5

decoder: whisper
decoder_conf:
    whisper_model: medium
    dropout_rate: 0.0

preprocessor: default
preprocessor_conf:
    whisper_language: "zh"
    whisper_task: "transcribe"

model_conf:
    ctc_weight: 0.0
    lsm_weight: 0.1
    length_normalized_loss: false
    extract_feats_in_collect_stats: false
    sym_sos: "<|startoftranscript|>"
    sym_eos: "<|endoftext|>"
    # do_pad_trim: true         # should be set when doing zero-shot inference
freeze_param: ["encoder","decoder"]

frontend: null
input_size: 1                   # to prevent build_model() from complaining

seed: 2022
log_interval: 100
num_att_plot: 0
num_workers: 0
sort_in_batch: descending       # how to sort data in making batch
sort_batch: descending          # how to sort created batches
batch_type: numel
batch_bins: 1200000         # good for 8 * RTX 3090 24G
accum_grad: 4
max_epoch: 40
patience: none
init: none
best_model_criterion:
-   - valid
    - acc
    - max
keep_nbest_models: 3

use_amp: true
cudnn_deterministic: false
cudnn_benchmark: false

# LoRA finetune related
use_adapter: true
adapter: lora
save_strategy: all
adapter_conf:
    rank: 8
    alpha: 16
    dropout_rate: 0.05
    target_modules: ["query", "key","value","token_embedding","attn.out"]

optim: adamw
grad_clip: 1.0
optim_conf:
    lr: 5.0e-04
    weight_decay: 0.01
    betas:
    - 0.9
    - 0.99
    eps: 1.0e-06
scheduler: warmuplr
scheduler_conf:
    warmup_steps: 1500
